src/components/addToCart/AddToCard.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { useContext, useEffect, useState } from 'react';
+import { setLocalStorageItem } from '../../helpers/util';
+import { addBasket, GlobalContext, removeProductInBasket } from '../../reducer';
+import { Product } from '../../types/product';
+import { ShoppingProduct } from '../../types/shoppingProduct';
+
+import './addToCart.scss';
+
+type Props = {
+  product: Product;
+};
+
+export const AddToCart: React.FC<Props> = ({ product }) => {
+  const [state, dispatch] = useContext(GlobalContext);
+  const [selected, setSelected] = useState(false);
+
+  const addProductToBasket = (item: Product) => {
+    if (!state.basketList.some((el: ShoppingProduct) => el.item === item)) {
+      dispatch({ type: addBasket, product: { item, value: 1 } });
+      setLocalStorageItem('shoppingList', [
+        ...state.basketList,
+        { item, value: 1 },
+      ]);
+    }
+  };
+
+  const remove = () => {
+    const list: ShoppingProduct[] | []
+      = JSON.parse(localStorage.getItem('shoppingList') as string) || [];
```

Comment:
could you create a helper to get a value from Local Storage?

Reply:
+